### PR TITLE
add "Walk Under Solid Hanging Bodies" from PrBoom+

### PR DESCRIPTION
**Pull request by fabiangreffrath**

### CODE REVIEW

**Review by rfomin — APPROVED** [review]
Looks good to me, but to be honest I don't know much about this issue. Maybe we should combine all options of this type into one option "map fixes".

**Comment by fabiangreffrath** [comment]
> Looks good to me, but to be honest I don't know much about this issue. Maybe we should combine all options of this type into one option "map fixes".

Hm, I don't think so. These changes are a bit too drastic, so I'd like to have them more fine-grained.

File: src/m_misc.c [hunk 1]
```diff
@@ -463,6 +464,13 @@ default_t defaults[] = {
     "1 to enable \"ghost monsters\" (resurrected pools of gore are translucent)"
   },
 
+  {
+    "hangsolid",
+    (config_t *) &hangsolid, NULL,
+    {0}, {0,1}, number, ss_enem, wad_no,
```
Comment (fabiangreffrath) [inline src/m_misc.c:470]: We have collected some unrelated config keys with `ss_enem` category again. Will have to clean this up (probably only after this PR, though).

Comment (rfomin) [reply src/m_misc.c:470]: Oh, I see. I always forget about it.

Comment (fabiangreffrath) [reply src/m_misc.c:470]: What would be the right category, btw?

Comment (rfomin) [reply src/m_misc.c:470]: > What would be the right category, btw?

`ss_gen` for "General" menu.

**Pull request merged**